cli/config_file.rs

```diff
@@ -266,10 +266,26 @@ impl Serialize for TsConfig {
   }
 }
 
+#[derive(Clone, Debug, Default, Deserialize)]
+#[serde(default)]
+pub struct LintRulesConfig {
+  pub tags: Option<Vec<String>>,
+  pub include: Option<Vec<String>>,
+  pub exclude: Option<Vec<String>>,
+}
+
+#[derive(Clone, Debug, Default, Deserialize)]
+#[serde(default)]
+pub struct LintConfig {
+  pub rules: LintRulesConfig,
+  pub ignore: Vec<String>,
+}
```

Comment:
I'm not happy with this schema, I'd much prefer to go with the solution proposed in: https://github.com/denoland/deno/issues/11686#issuecomment-898863720

In this schema there's no way to add files that should be included.

Reply:
I think at least changing this to `"files": { "exclude": [...] }` might help keep this flexible for future support of "includes" and won't confuse people about what "ignore" means.

Reply:
Inclusion and exclusion of files is already possible in CLI using positional arg and `--ignore` option respectively. The schema would schema slightly different names but would support exactly same functionality. So I'm highly interested favor of using `files`.

Reply:
I changed the schema to use `"files": { "include": [], "exclude": {} }`

Reply:
Does exclude have the same semantics as ignore here? would be better to retain the name imho so there's no new things to know (also we've used include and exclude for regular expression filters in coverage cli).

Just throwing out a strawman:
```ts
"files": {
  "collect": [],
  "ignore": " [],
 }
 ```

Reply:
I think it's better to just follow `tsconfig.json` here as a lot of developers should be familiar with that, and it uses `exclude` and `include` for files.